presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkQueryExecutionFactory.java

```diff
@@ -426,6 +435,8 @@ private static QueryInfo createQueryInfo(
                 succinctBytes(peakTaskUserMemoryInBytes),
                 succinctBytes(peakTaskTotalMemoryInBytes));
 
+        Optional<ResourceGroupId> resourceGroupId = sparkQueueName.map(name -> new ResourceGroupId(Arrays.asList(name.split("\\."))));
```

Comment:
Do you think it is necessary to split it on `"\\."`? Shouldn't we just store it as is in a single fragment?

Reply:
@arhimondr : Many years ago there is a discussion with @raghavsethi  and we agreed "." shouldn't be a valid character in resource group name :) ... 

anyhow, assume it doesn't cause trouble, I am happy to just store it in a single fragment.